apps/prairielearn/src/pages/news_item/news_item.html.ts

```diff
@@ -0,0 +1,78 @@
+import { formatDate } from '@prairielearn/formatter';
+import { html, unsafeHtml } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+import type { NewsItem } from '../../lib/db-types.js';
+
+export function NewsItem({
+  resLocals,
+  newsItem,
+  newsItemHtml,
+  userIsInstructor,
+}: {
+  resLocals: Record<string, any>;
+  newsItem: NewsItem;
+  newsItemHtml: string;
+  userIsInstructor: boolean;
+}) {
+  const { urlPrefix } = resLocals as { urlPrefix: string };
+  const formattedDate = formatDate(
+    newsItem.date,
+    resLocals.course_instance?.display_timezone ?? resLocals.course?.display_timezone ?? 'UTC',
```

Comment:
Not requesting a change since this matches existing behavior, but it feels very very weird to conditionally use a course or course instance timezone to render dates that aren't in any way tied to a course or course instance.

Reply:
Using UTC might confuse some people that assume everything is in their own timezone. That said, I don't think this is significant since only dates are shown, and having the news date be one day off will probably not cause any problems. I don't have a strong reason in either direction, up to you.

Reply:
Given that we only show the date, I'd be happy to always render it in UTC?

Reply:
Done in 59cf7c3a7.